src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
 }
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.card__image {
+  background-image: url("../images/imac.jpeg");
+  background-size: cover;
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+  text-align: left;
+  color: #060b35;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  display: flex;
+  justify-content: space-between;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 5px;
+  color: #616070;
+  font-weight: 400;
```

Comment:
`font-weight: 400;`  is a default browser value. Remove from similar cases as well
```suggestion
```